src/deadline_worker_agent/sessions/attachment_models.py

```diff
@@ -135,11 +139,13 @@ def from_dict(cls, data: Dict[str, Any]) -> "WorkerManifestProperties":
             outputRelativeDirectories=manifest_props_data.get("outputRelativeDirectories"),
         )
 
-        return cls(
+        worker_props = cls(
             manifest_properties=manifest_properties,
             local_manifest_paths=data.get("localManifestPaths", []),
             local_root_path=data["localRootPath"],
         )
+        worker_props.local_input_manifest_path = data.get("localInputManifestPath")
+        return worker_props
```

Comment:
nit: We updated the `__init__` to take the new parameter, can we just add to the argument list instead of setting the field in a separate statement?

```suggestion
        return cls(
            manifest_properties=manifest_properties,
            local_manifest_paths=data.get("localManifestPaths", []),
            local_root_path=data["localRootPath"],
            local_input_manifest_path = data.get("localInputManifestPath")
        )
```